### ISSUE TITLE
:Unite taskwarrior/reports

### ISSUE THREAD

**Issue by linuxcaffe** [blakesweeney/unite-taskwarrior#27]
This is a sub-set of taskwarrior/cmd, but also includes the text of 

report.foo.description='This is the report on the Foo project"

**Comment by blakesweeney** [comment]
So you'd like a source that lists known reports and then lets you run them? I'm looking the reports a bit and I might be able to do something like that, but we might lose the interactivity from unite. Not sure how I feel about that. Could you give some more details of what you expect?

**Comment by linuxcaffe** [comment]
Yeah, what I'm thinking of for taskwarrior/reports is very much like taskwarrior/cmd, but a) only shows reports, and b) shows the report descriptions. I'm thinking that the actions between /cmd and /reports should be very similar. 

Once implemented, and working nice, it might be good to elide reports from the cmd list, reducing some redundancy.
